### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by YorkAARGH**
**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**

**Semantic versioning classification:**

- [X] This PR changes the framework's interface (methods or parameters added)
  - [X] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

Removed the installer and all required packages.
Removed the `loadCommand` and `unloadCommand` functions.
Removed the `wait` function, was no longer used.
Removed the `owners` functionality as the concept was too advanced for an example bot.
Moved the discord bot token into an `.env` file for better security.
Added `owner` to the `.env` file.
Required the `logger` file where it was needed instead of binding it to `client`.
Required the `settings` database and `config` file where needed.
Created a `container` property to attach `commands`, `aliases`, `slashcmds` and `levelCache` to.
Moved the `clean` function into the `eval` command.
Replaced `chalk` with `colorette`.
Replaced `moment` and `moment-duration-format` with `@sapphire/time-utilities`

Updated all of the code to reflect these changes.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,56 +2,28 @@
...
-try {
-  require("./config.js");
-} catch (e) {
-  console.log("Creating a config.js file for the bot");
-  require("./setup.js");
-}
+require("dotenv").config();
 
 // Load up the discord.js library
 const { Client, Collection} = require("discord.js");
```
Comment (GoldenAngel2) [inline index.js]:
```suggestion
const { Client, Collection } = require("discord.js");
```

File: index.js [hunk 2]
```diff
@@ -2,56 +2,28 @@
...
+const { intents, partials, permLevels } = require("./config.js");
+const logger = require("./modules/Logger.js");
 // This is your client. Some people call it `bot`, some people call it `self`,
 // some might call it `cootchie`. Either way, when you see `client.something`,
 // or `bot.something`, this is what we're referring to. Your client.
 const client = new Client({
-  intents: config.intents,
-  partials: config.partials
+  intents: intents,
+  partials: partials
```
Comment (GoldenAngel2) [inline index.js]: couldn't this line be `intents, partials` instead of 
```js
  intents: intents,
  partials: partials
```

Comment (YorkAARGH) [reply index.js]: Ooh, very good point! I'll fix em

**Comment by jaspermayone** [comment]
@YorkAARGH When is this going to be merged?

**Comment by jaspermayone** [comment]
(not to sound impatient) 😄

**Comment by YorkAARGH** [comment]
> @YorkAARGH When is this going to be merged?

When it's finished, there's a few things that remain to be done, then I'll need people to test it to eliminate the whole "It works on my machine" situation.

**Comment by jaspermayone** [comment]
Hey @YorkAARGH, I think we should keep the owner only function. Personally, I think that is something that people can learn from and dissect.

**Comment by YorkAARGH** [comment]
> Hey @YorkAARGH, I think we should keep the owner only function. Personally, I think that is something that people can learn from and dissect.

I was initially in the same boat, but we don't cover teams at all in the guide and it wasn't really explained well, if at all in the code.

The team functionality (to me at least) would imply using this example code in a large production based project, which we absolutely do not want to encourage as it lacks significant features that would be able to handle such a scale, which is why it was removed in the first place.

**Comment by akii0008** [comment]
Not to be a pain but if you're gonna bump this up one major semver, you may want to include support for discord's slash commands as that's where the bot ecosystem seems to be headed

**Comment by jaspermayone** [comment]
Yay! 🎉

**Pull request merged**